divi/src/txdb.h

```diff
@@ -8,16 +8,12 @@
 #define BITCOIN_TXDB_H
 
 #include "leveldbwrapper.h"
-#ifndef BITCOIN_COINS_H
 #include <coins.h>
```

Comment:
coins.h is included in both txdb.h and txdb.cpp, and thus redundant. Is it possible to remove coins.h from the header by forward declaring all the necessary types?

Reply:
You mentioned a few types needed in:
https://github.com/galpHub/Divi/pull/22#discussion_r463355931
Only the CCoins in the CCoinsCacheEntry seems to care about the CCoins type directly. I suppose we can address this in another PR since we need to extract the individual classes in coins.h